fashion-messaging-app-ui/src/components/App/App.jsx

```diff
@@ -1,39 +1,43 @@
-import "./App.css"
+import "./App.css";
 import Search from "../Search/Search";
 import Navbar from "../Navbar/Navbar";
 import FashionItems from "../FashionItems/FashionItems";
 import { useEffect, useState } from "react";
 import { createClient } from "pexels";
 
-const API_KEY = "06ZcwjgbmJBM8T2TxLUZ5iwdXXGxiAgz0Z018b7QPKwR1ExipkFjaAuw"
+const API_KEY = "06ZcwjgbmJBM8T2TxLUZ5iwdXXGxiAgz0Z018b7QPKwR1ExipkFjaAuw";
 const clientAPI = createClient(API_KEY);
-const query = "Fashion";
+const defaultQuery = "Fashion";
 const PHOTOS = 100;
 
 export default function App() {
   const [photos, setPhotos] = useState([]);
+  const [searchQuery, setSearchQuery] = useState("");
+
   useEffect(() => {
-    clientAPI.photos.search({ query, per_page: PHOTOS })
+    const query = searchQuery || defaultQuery;
```

Comment:
does this just set query to searchQuery if it exists and defaultQuery if it hasn't been set?